### PR TITLE
Removal of Nazi items

### PR DESCRIPTION
**Pull request by r4d6**
##  About The Pull Request

Removing the Nazi Hardsuit & Hardsuit helmet code and sprite.

## Why It's Good For The Game

According to Citadel's CONTRIBUTING guide, we can't have nazi shit in the game.

> Do not add any of the following in a Pull Request or risk getting the PR closed:
- **National Socialist Party of Germany content, National Socialist Party of Germany related content, or National Socialist Party of Germany references**
- Code where one line of code is split across mutiple lines (except for multiple, separate strings and comments; in those cases, existing longer lines must not be split up)
- Code adding, removing, or updating the availability of alien races/species/human mutants without prior approval. Pull requests attempting to add or remove features from said races/species/mutants require prior approval as well.
- Just because something isn't on this list doesn't mean that it's acceptable. Use common sense above all else.
Source : https://github.com/Citadel-Station-13/Citadel-Station-13/blob/master/.github/CONTRIBUTING.md (At the bottom)

Also I was told that apparently Github bans any mention of nazis. So better safe than sorry

## Changelog
:cl:
del: Removed Nazi Clothing
del: Removed Nazi Hardsuit
del: Removed Nazi Gun
del: Removed Nazi Vendor
del: Removed Nazi Clothing Sprites
/:cl:

### CODE REVIEW

**Comment by lolman360** [comment]
that's actually a holdover from /tg/'s guidelines, citadel hasn;t bothered to change it

**Comment by r4d6** [comment]
I did not know that

File: code/modules/projectiles/guns/energy/plasma_cit.dm [hunk 1]
```diff
@@ -32,16 +32,6 @@
 	ammo_type = list(/obj/item/ammo_casing/energy/plasmagun/light)
 	ammo_x_offset = 2
 
-
-/obj/item/gun/energy/plasma/MP40k
```
Comment (deathride58) [inline code/modules/projectiles/guns/energy/plasma_cit.dm:36]: There's no need to remove this.

**Review by deathride58 — CHANGES_REQUESTED** [review]

**Comment by r4d6** [comment]
Probably an error

**Comment by Linzolle** [comment]
> 
> 
> that's actually a holdover from /tg/'s guidelines, citadel hasn;t bothered to change it

even if it is a holdover it's.... a very rightfully placed holdover

File: code/modules/projectiles/guns/energy/plasma_cit.dm (hunk 1 above)
Comment (r4d6) [reply code/modules/projectiles/guns/energy/plasma_cit.dm:36]: How to I change it?

**Comment by Weblure** [comment]
Deleting 11 items for vague association with WWII Germany is overkill. I'm alright with changing any tasteless item descriptions that were grandfathered in, but a lot of these items otherwise have no direct correlation with Nazis.

**Comment by r4d6** [comment]
> Deleting 11 items for vague association with WWII Germany is overkill. I'm alright with changing any tasteless item descriptions that were grandfathered in, but a lot of these items otherwise have no direct correlation with Nazis.

Vague association with WWII Germany
Hardsuit named "Nazi Hardsuit"
Vender named "NaziVend"
"Vague" association

**Comment by Weblure** [comment]
> > Deleting 11 items for vague association with WWII Germany is overkill. I'm alright with changing any tasteless item descriptions that were grandfathered in, but a lot of these items otherwise have no direct correlation with Nazis.
> 
> Vague association with WWII Germany
> Hardsuit named "Nazi Hardsuit"
> Vender named "NaziVend"
> "Vague" association

The physical items themselves, not their names and descriptions, which I'm fine with changing, as already stated. 

Seriously, what's "Nazi" about the hardsuit other than it being named that? Pretty sure WWII Germany didn't march around in fictional space suits bearing the Nazi symbol (which _none_ of these items bear).

**Comment by Keekenox** [comment]
I can't believe anyone was disagreeing with the removal of the dumb broken nazi shit you guys are wild lmao.

**Comment by Weblure** [comment]
> I can't believe anyone was disagreeing with the removal of the dumb broken nazi shit you guys are wild lmao.

This wasn't the right way to do it. They were straight up removing the plasma MP40 from the game, which is a balance change, along with items that have no nazi correlation, other than it saying 'nazi' in the code, which can't be seen ingame.

**Pull request closed without merging**